config/locales/en.yml

```diff
@@ -540,7 +540,7 @@ en:
         documentation_hosted_url: '%{docs_base_url}/deployment-options/apicast-hosted'
         documentation_self_managed_url: '%{docs_base_url}/deployment-options/apicast-self-managed-nginx-config-files'
         documentation_oauth_url: '%{docs_base_url}/deployment-options/apicast-oauth-before-may-2017'
-        documentation_response_codes_tracking_url: '%{docs_base_url}/admin_portal_guide/response-codes-tracking'
+        documentation_response_codes_tracking_url: '%{docs_base_url}/admin_portal_guide/setting-up-and-evaluating-the-threescale-response-codes-log-for-your-api_analytics-for-threescale-apis'
```

Comment:
I think we need to change the text where this link is referenced also.

It's now
```
Our HowTo tutorial on [Response Codes Tracking](https://access.redhat.com/documentation/en-us/red_hat_3scale/2-saas/html/admin_portal_guide/response-codes-tracking) on the 3scale support portal
```
I'd change it to something like:
```
Documentation on [response codes logging](https://access.redhat.com/documentation/en-us/red_hat_3scale/2-saas/html/admin_portal_guide/setting-up-and-evaluating-the-threescale-response-codes-log-for-your-api_analytics-for-threescale-apis) on the Red Hat Customer Portal.
```

Reply:
Preferably we would remove the double occurrence of the word "on". So I suggest this:

`Documentation about response code logging on the Red Hat Customer Portal.`